### PR TITLE
[MXAPPS-987] Add MMS and Scala to Benchmark.AI reports

### PR DESCRIPTION
**Pull request by vishaalkapoor**
Add the two suites of benchmarks (updating the MMS benchmarks).

### CODE REVIEW

File: reporting/config/benchmarks.yaml [hunk 1]
```diff
@@ -307,109 +307,138 @@ benchmarks:
...
-    Metric Suffix: nightly.p3_8x
+    Framework Desc: MMS-GPU
+    Model: Resnet-18
+    Benchmark Desc:
+    Instance Type: p3.16xlarge
+    Error Rate: throughput_resnet-18_Inference_Request_aggregate_report_error
+    Throughput: throughput_resnet-18_Inference_Request_Throughput
+    Latency: latency_resnet-18_Inference_Request_Average
+    P50 Latency: latency_resnet-18_Inference_Request_Median
+    # TODO: A corresponding metric should be added for the P90.
```
Comment (andrewfayres) [inline reporting/config/benchmarks.yaml]: NP: I think we can remove this TODO.

File: reporting/config/benchmarks.yaml [hunk 2]
```diff
@@ -307,109 +307,138 @@ benchmarks:
...
+    Error Rate: throughput_noop_Inference_Request_aggregate_report_error
+    Throughput: throughput_noop_Inference_Request_Throughput
+    Latency: throughput_noop_Inference_Request_Average
+    P50 Latency: throughput_noop_Inference_Request_Median
+    P90 Latency: throughput_noop_Inference_Request_aggregate_report_90_line
+    P99 Latency: throughput_noop_Inference_Request_aggregate_report_99_line
+    Latency: latency_noop_Inference_Request_Average
+    P50 Latency: latency_noop_Inference_Request_Median
+    P90 Latency: latency_noop_Inference_Request_aggregate_report_99_line
+    P99 Latency: latency_noop_Inference_Request_aggregate_report_error
```
Comment (andrewfayres) [inline reporting/config/benchmarks.yaml]: Lines 411-414 look to be duplicates of 406-410 (Latency through P99)

Comment (vishaalkapoor) [reply reporting/config/benchmarks.yaml]: Yes, good catch. The throughput lines are overridden by the latency lines which ultimately is the correct behavior, but lines 408-410 can be removed (they're mentioned in comment as unsupported for now). I'll remove 408-410.

File: reporting/config/benchmarks.yaml (hunk 1 above)
Comment (vishaalkapoor) [reply reporting/config/benchmarks.yaml]: Thanks - I added that TODO and found the P90 metric name but didn't remove the TODO. :)

**Comment by vishaalkapoor** [comment]
Thanks @andrewfayres. I made the updates. If the config looks good, I'll check it in.

**Comment by vishaalkapoor** [comment]
Thanks @andrewfayres !

**Pull request merged**